app/src/main/res/values/strings.xml

```diff
@@ -204,8 +204,8 @@
     <string name="export_data_title">Export database</string>
     <string name="clear_cookie_title">Clear reCAPTCHA cookies</string>
     <string name="recaptcha_cookies_cleared">reCAPTCHA cookies have been cleared</string>
-    <string name="import_data_summary">Overrides your current history and subscriptions</string>
-    <string name="export_data_summary">Export history, subscriptions and playlists</string>
+    <string name="import_data_summary">Overrides everything except download history (and settings if you don\'t import them)</string>
+    <string name="export_data_summary">Export everything except download history</string>
```

Comment:
I would explicitly name everything that gets exported like it was before.

If in the future something new gets added, where one could assume that it is exportable, but is not actually exported, this string would cause confusion.

Reply:
Good point. Anybody has a full list?

Reply:
```
sqlite> .tables
android_metadata              remote_playlists            
feed                          room_master_table           
feed_group                    search_history              
feed_group_subscription_join  stream_history              
feed_last_updated             stream_state                
playlist_stream_join          streams                     
playlists                     subscriptions
```
So video history and stream states, search history, subscriptions, subscription groups, the last updated feed, playlists. Additionally settings, which are not in the database file.
I think all of the above can be condensed into history, subscriptions, playlists and settings.

Reply:
Changed.